Public/Src/FrontEnd/UnitTests/Script.ErrorHandling/LintRules/TestEnforceAmbientAccessInConfig.cs

```diff
@@ -29,6 +29,7 @@ public TestEnforceAmbientAccessInConfig(ITestOutputHelper output) : base(output)
         [InlineData("Context.hasMount(\"foo\")")]
         [InlineData("Context.getTemplate()")]
         [InlineData("Context.getBuildEngineDirectory(\"foo\")")]
+        [InlineData("File.readPipGraphFragment(\"foo\", f`package.dsc`, [])")]

```

Comment:
>File [](start = 21, length = 4)

Transformer

Reply:
Done

---
In reply to: [289154259](https://github.com/ljcollins25/BuildXL/pull/1#discussion_r289154259) [](ancestors = 289154259)